src/PKSim.Presentation/Presenters/Charts/EditTimeProfileAnalysisChartPresenter.cs

```diff
@@ -180,10 +239,34 @@ private void showAnalysis()
          _timeProfileAnalysisChartView.ShowPKAnalysisView();
       }
 
+      private bool populationDataCollectorSupportsDifferentAggregations(IPopulationDataCollector populationDataCollector)
+      {
+         return populationDataCollector is PopulationSimulation;
+      }
+
+      private IEnumerable<QuantityPKParameter> extractPKParameters(PopulationSimulation populationSimulation)
+      {
+         if (populationSimulation == null)
```

Comment:
can it be null here? private method only used after the cast.

Reply:
Right, already checked at `if (!populationDataCollectorSupportsDifferentAggregations(PopulationDataCollector))`